tests/accel/upgrade/internal/accelparams/vars.go

```diff
@@ -0,0 +1,45 @@
+package accelparams
+
+import (
+	"fmt"
+	"slices"
+	"time"
+
+	"github.com/openshift-kni/eco-gotests/tests/accel/internal/accelparams"
+	"github.com/openshift-kni/k8sreporter"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var (
+	// Labels represents the range of labels that can be used for test cases selection.
+	Labels = []string{accelparams.Label, Label}
+	// IbuWorkloadImage is the test workload image.
+	IbuWorkloadImage = `registry.redhat.io/openshift4/ose-hello-openshift-rhel8@sha256:10dca31348f07e1bfb56ee93c324525cceefe27cb7076b23e42ac181e4d1863e`
```

Comment:
Looks like this image will be changed periodically so I would recommend to make it environment variable instead of changing code after each image update.